src/Sylius/Component/Core/spec/Model/CustomerSpec.php

```diff
@@ -64,4 +66,39 @@ function it_adds_address_when_billing_address_is_set(AddressInterface $address):
         $this->setDefaultAddress($address);
         $this->hasAddress($address)->shouldReturn(true);
     }
+
+    function it_has_no_user_by_default(): void
+    {
+        $this->getUser()->shouldReturn(null);
+    }
+
+    function its_user_is_mutable(ShopUserInterface $user): void
+    {
+        $user->setCustomer($this)->shouldBeCalled();
+
+        $this->setUser($user);
+        $this->getUser()->shouldReturn($user);
+    }
+
+    function it_throws_an_invalid_argument_exception_when_user_is_not_a_shop_user_type(UserInterface $user)
+    {
+        $this->shouldThrow(\InvalidArgumentException::class)->during('setUser', [$user]);
+    }
+
+    function it_resets_customer_of_previous_user(ShopUserInterface $previousUser, ShopUserInterface $user)
+    {
+        $this->setUser($previousUser);
+
+        $previousUser->setCustomer(null)->shouldBeCalled();
+
+        $this->setUser($user);
+    }
+
+    function it_does_not_replace_user_if_it_is_already_set(ShopUserInterface $user)
```

Comment:
Missing return `: void` for this and some other functions :)

Reply:
> Missing return `: void` for this and some other functions :)

Fixed! thanks